youtube_transcript_api/test/test_cli.py

```diff
@@ -131,7 +131,7 @@ def test_argument_parsing__only_video_ids(self):
         self.assertEqual(parsed_args.languages, ["en"])
 
     def test_argument_parsing__video_ids_starting_with_dash(self):
-        parsed_args = YouTubeTranscriptCli("\-v1 \-\-v2 \--v3".split())._parse_args()
+        parsed_args = YouTubeTranscriptCli("-v1 --v2 --v3".split())._parse_args()
```

Comment:
The point of this test is to check whether the CLI allows for escaping leading `-` and `--` by using `\`. So by removing those from the input string, you've effectively made this test pointless!

However, you can use a raw string here to get rid of the PEP-8 warning:
```suggestion
        parsed_args = YouTubeTranscriptCli(r"\-v1 \-\-v2 \--v3".split())._parse_args()
```
```